test/java/seedu/addressbook/commands/DeleteCommandTest.java

```diff
@@ -21,6 +22,7 @@
 import seedu.addressbook.ui.TextUi;
 import seedu.addressbook.util.TestUtil;
 
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Not necessary since leaving it out will default back to JUnit 4 behavior. 

However, there are some nice benefits to having it in as described [here](https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle) and it has the nice side effect of making the tests go quite a bit faster.